app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,2 @@
+class Users::SessionsController < Devise::SessionsController
+end
```

Comment:
You can remove this file entirely if you remove the hash argument in the `devise_for` in `routes.rb`.